npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -198,7 +309,7 @@ def handle_npm_updater_error(error, lockfile)
           # workspace project)
           sub_dep_local_path_error = "does not contain a package.json file"
           if error_message.match?(INVALID_PACKAGE) ||
-             error_message.start_with?("Invalid package name") ||
+             error_message.include?("Invalid package name") ||
```

Comment:
Note: npm 7 now exits with a full error log so we can't do starts_with? on anything meaningful